packages/ui/src/components/va-alert/VaAlert.demo.vue

```diff
@@ -50,7 +50,7 @@
         />
         Better check yourself, you're not looking too good.
       </va-alert>
```

Comment:
VaSidebar demo is broken.

It is issue from our release v1.4.0 branch. useTextColor must be good solution to resolve this issue. 

![image](https://user-images.githubusercontent.com/23530004/168286586-c268d6b3-769c-4ead-9e6f-7df93640e0ce.png)